README.md

```diff
@@ -1 +1,106 @@
-# cairo-annotations
+# Cairo Annotations
+
+The `cairo-annotations` crate provides tools tailored for working with annotations in the Cairo language.
+These
+annotations are part of the Sierra Debug Information format and serve to enrich the code by detailing information.
+
+> 📝 **Note**
+>
+> Although this crate is primarily utilized by projects like `cairo-coverage`, `cairo-profiler`, and `starknet-foundry`,
+> it is also fully capable of functioning as a standalone library to work with Sierra Debug Information annotations.
+
+## Features of Cairo Annotations
+
+### Structured Annotations
+
+`cairo-annotations` offers a structured representation, allowing for more ergonomic manipulation of annotations.
+Some
+key features include:
```

Comment:
```suggestion
Some key features include:
```

I can see there are multiple places having a newline whitespace after first word. Is this something intended?

Reply:
No, it seems like a mistake with formatter 😅. I will fix it